scripts/release.sh

```diff
@@ -13,64 +13,70 @@ if [[ $# != 2 ]] && [[ $# != 3 ]]; then
     echo 'then this script runs "make deploy" after sourcing "environment.{DEST}",'
     echo "where DEST is the destination branch."
     echo
-    echo "If the --force flag is given, deployment will proceed even if CI checks fail."
+    echo "If ${DSS_FORCE_RELEASE}=='force', deployment will proceed even if CI checks fail."
     echo
-    echo "Usage: $(basename $0) source_branch dest_branch [--force]"
+    echo "Usage: $(basename $0) source_branch dest_branch"
     echo "Example: $(basename $0) master staging"
     exit 1
 fi
 
-echo "Please review and confirm your active AWS account configuration:"
-aws configure list
-aws sts get-caller-identity
-aws iam list-account-aliases
-echo "Is this correct?"
-select result in Yes No; do
-    if [[ $result != Yes ]]; then exit 1; else break; fi
-done
+if [[ "hca_cicd" != $(whoami) ]]; then
+    # Skip when this script is executed by the GitLab runner
+    echo "Please review and confirm your active AWS account configuration:"
+    aws configure list
+    aws sts get-caller-identity
+    aws iam list-account-aliases
+    echo "Is this correct?"
+    select result in Yes No; do
+        if [[ $result != Yes ]]; then exit 1; else break; fi
+    done
+fi
```

Comment:
I don't think this is a good pattern. It intertwines the concerns. What if I sudo to impersonate this user for some reason? Consider just calling this script with `yes |`:
```bash
yes Yes | scripts/release.sh
```

Reply:
Fixed.